frontend/pages/ManageControlsPage/Scripts/components/DeleteScriptModal/DeleteScriptModal.tsx

```diff
@@ -9,7 +12,7 @@ interface IDeleteScriptModalProps {
   scriptName: string;
   scriptId: number;
   onCancel: () => void;
-  onDelete: (scriptId: number) => void;
+  onDelete: () => void;
```

Comment:
What do you think about naming this `onDone` or something to distinguish it from the `onClickDelete` function below?

Reply:
makes sense. im gonna change it to `onDeleted`.